servers/lib/dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:20.10.0-slim
```

Comment:
When I build this dockerfile outside of compose, add to images and then run it, the container does not want to stop.

```bash
docker run -p 4001:4001 --volume ${PWD}/deploy/config/lib.docker:/dtaas/libms/.env --volume ${PWD}/files:/dtaas/libms/files prasadtalasila/dtaas-libms:latest
```

The corresponding dockerfile
https://hub.docker.com/layers/prasadtalasila/dtaas-libms/latest/images/sha256-b2948603c44b004b0cc69ddb4c7b22eca32d691eff29aaa7955db08c79f00366?context=repo

Reply:
So when you click/run "stop" on the container it doesn't work?

Reply:
Might have to increase the memory docker has available

Reply:
It works for me, but takes a bit of time